File: Python/1115 - Quadrant.py
stop = False
while (stop != True):
    s = input().split()
    for i in range(len(s)):
        s[i] = int(s[i])
    if (s[0] == 0) or (s[1] == 0):
        stop = True
    else:
        if (s[0] > 0):
            if (s[1] > 0):
                print("primeiro")
            else:
                print("quarto")
        else:
            if (s[1] > 0):
                print("segundo")
            else:
                print("terceiro")

File: C++/1115 - Quadrant.cpp
#include <iostream>

void quadrante(int x, int y)
{
    if (x > 0)
    {
        if (y > 0)
        {
            std::cout << "primeiro\n";
        }
        else if (y < 0)
        {
            std::cout << "quarto\n";
        }
    }
    else if (x < 0)
    {
        if (y > 0)
        {
            std::cout << "segundo\n";
        }
        else if (y < 0)
        {
            std::cout << "terceiro\n";
        }
    }
}

int main() {
    int x, y;
    do
    {
        std::cin >> x >> y;
        if (x == 0 || y == 0)
        {
            break;
        }
        quadrante(x,y);
    }
    while (x != 0 && y != 0);
    return 0;
}